### PR TITLE
Slider and Product view section 

### PR DESCRIPTION
**Pull request by wali39**
slider both

From PSD:
add to cart

With  Code:
Product view

I have completed my section please see it.

### CODE REVIEW

File: src/components/productView/ProductView.module.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Cantata+One&family=Montserrat:wght@400;600;700&display=swap");
+/******* Product custom css start *******/
+* {
+  font-family: "Montserrat";
+}
+.Img1 {
```
Comment (alamiiin) [inline src/components/productView/ProductView.module.css]: why did you write class names in PascalCase? They should be camelCase.

File: src/components/Slider/Slider.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import React, { Component} from "react";
```
Comment (alamiiin) [inline src/components/Slider/Slider.js]: Fix foldername

File: src/components/productView/ProductView.module.css (hunk 1 above)
Comment (wali39) [reply src/components/productView/ProductView.module.css]: className casing problem fixed

File: src/components/Slider/Slider.js (hunk 2 above)
Comment (wali39) [reply src/components/Slider/Slider.js]: Folder name casing problem fixed

**Review by alamiiin — APPROVED** [review]

**Pull request merged**